scripts/test/system.sh

```diff
@@ -47,6 +47,43 @@ function test_subctl_gather() {
     echo "::endgroup::"
 }
 
+function check_service_exported() {
+    status=$(kubectl get serviceexport nginx-demo --context "$cluster" -n default -o=jsonpath='{.status.conditions[?(@.type=="Valid")].status}')
+    if [[ "$status" == "True" ]]; then
+        return 0
+    fi
+
+    c=$(kubectl get serviceexport nginx-demo --context "$cluster" -n default -o=jsonpath='{.status.conditions[?(@.type=="Valid")]}')
+    echo "Service not exported: $c"
+
+    return 1
+}
+
+function export_service() {
+    cat <<EOF | kubectl --context "$cluster" apply -f -
```

Comment:
```suggestion
    kubectl apply -f - <<EOF
```

The context is automatically set by Shipyard since you used `with_context`.
The here-doc can be passed straight to the command, no need to pass it through `cat`.